src/lemlib/devices/encoder/rotation.cpp

```diff
@@ -33,7 +33,7 @@ float RotationEncoder::getAngle() {
  * Reset/calibrate the optical encoder
  */
 bool RotationEncoder::reset() {
-    lastAngle = 0;
+    lastAngle = 0_deg;
```

Comment:
```suggestion
    lastAngle = 0_rad;
```
Change to radians for consistency